main.py

```diff
@@ -56,10 +62,12 @@ def exchange_token():
     role = request.args.get('role')
     role_mapping = config['roleMapping'].get(role)
     if not role_mapping:
+        app.logger.debug({'message': 'Unknown role', 'role': role_mapping})
```

Comment:
I think that this should be error level logging

Reply:
Generally all 400 messages should be logged as error.

Reply:
Yes, you're absolutely right. It should be fixed by now 😉